common/djangoapps/track/tests/test_shim.py

```diff
@@ -216,3 +221,67 @@ def test_multiple_backends(self):
             'timestamp': FROZEN_TIME,
         }
         assert_events_equal(expected_event, log_emitted_event)
+
+
+SequenceDDT = namedtuple('SequenceDDT', ['action', 'tab_count', 'current_tab', 'legacy_event_type'])
+
+
+@ddt.ddt
+class PrefixedEventProcessorTestCase(EventTrackingTestCase):
+    """
+    Test PrefixedEventProcessor
+    """
+
+    @ddt.data(
+        SequenceDDT(action=u'next', tab_count=5, current_tab=3, legacy_event_type=u'seq_next'),
+        SequenceDDT(action=u'next', tab_count=5, current_tab=5, legacy_event_type=None),
+        SequenceDDT(action=u'previous', tab_count=5, current_tab=3, legacy_event_type=u'seq_prev'),
+        SequenceDDT(action=u'previous', tab_count=5, current_tab=1, legacy_event_type=None),
+    )
+    def test_sequence_linear_navigation(self, sequence_ddt):
+        event_name = u'edx.ui.lms.sequence.{}_selected'.format(sequence_ddt.action)
+
+        event = {
+            u'name': event_name,
+            u'event': {
+                u'current_tab': sequence_ddt.current_tab,
+                u'tab_count': sequence_ddt.tab_count,
+                u'id': u'ABCDEFG',
+            }
+        }
+
+        process_event_shim = PrefixedEventProcessor()
```

Comment:
Not yet.  @mulby  suggested bokchoy tests.  That's probably the place to do it.  I don't think we can do it properly at the unittest level, because it depends on django settings, so we're really testing if the processor is registered in the settings, which is more of an integration-level question.